src/app/components/modules/Header.test.js

```diff
@@ -1,11 +1,18 @@
 /*global describe, it, before, beforeEach, after, afterEach */
 import React from 'react';
-import { expect } from 'chai';
-import { shallow } from 'enzyme';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15';
```

Comment:
If we add a `jestSetup.js` file per this commit  then we will not have to import the Adapter in every test...
https://github.com/steemit/condenser/commit/4f233a29ad734b4bccb13acd0937ef590bed8786

Reply:
ooh good call, will `cherry-pick`

Reply:
actually i don't think we should run these before every test -- they are only necessary for component tests & will slow down unit tests

Reply:
Sounds good.